### PR TITLE
Update CONTRIBUTING.md with Installation Steps

### PR DESCRIPTION
**Pull request by ameypv-rudder**
**Fixes** # (*issue*)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,17 +1,70 @@
...
+
+ ```
+ createdb jobsdb
+ createuser --superuser rudder
+ psql "jobsdb" -c "alter user rudder with encrypted password 'rudder'";
+ psql "jobsdb" -c "grant all privileges on database jobsdb to rudder";
+
+ ```
+
+4. Go to the [RudderStack dashboard] (https://app.rudderlabs.com/signup) and set up your account. Copy your workspace token from the top of the home page.
```
Comment (arnab-p) [inline CONTRIBUTING.md:34]: Let's change the link to https://app.rudderstack.com/signup ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,17 +1,70 @@
...
+
+5. Clone the RudderStack server repository. Run `git submodule init` and `git submodule update` to fetch the **rudder-transformer** repository. Then, navigate to the transformer directory using the following command:
+
+```
+cd rudder-transformer
+```
+6. Install dependencies using the command `npm i` and start the destination transformer using the following command:
+```
+node destTransformer.js
```
Comment (arnab-p) [inline CONTRIBUTING.md:43]: I think `npm start` sounds more familiar

**Review by arnab-p — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (ameypv-rudder) [reply CONTRIBUTING.md:34]: Agreed. Will incorporate this change.

File: CONTRIBUTING.md (hunk 2 above)
Comment (ameypv-rudder) [reply CONTRIBUTING.md:43]: Will make this change, thanks!

**Review by chandumlg — APPROVED** [review]

**Review by arnab-p — APPROVED** [review]

**Pull request merged**